sklearn/multiclass.py

```diff
@@ -968,25 +968,16 @@ class OutputCodeClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
-    feature_names_in_ : ndarray of shape (`n_features_in_`,)
+    feature_names_in_ : array-like of shape (n_features_in_,)
```

Comment:
I think that you need to keep the backticks. Otherwise, `sphinx` will think that `n_features_in_` should be hyperlinked with something because it finishes with an underscore.

Reply:
I didn't know that, thanks for the heads up.

Related to that, when do we use backticks in the documentation?
I have seen them used for variables, functions and snippets of code in the docstrings.

Sometimes there is just one backstick and other times there are two backticks. What is the difference?

Reply:
> Related to that, when do we use backticks in the documentation?
I have seen them used for variables, functions and snippets of code in the docstrings.

We are really inconsistent. IMO, we should use it whenever we refer to a variable. For functions and classes, we should favour the sphinx directive `:func:`, `:class:`, `:meth:` when it is possible.

> Sometimes there is just one backtick and other times there are two backticks. What is the difference?

In the past, we were forced to use double backticks to have the expected results. However, a couple of years ago, we changed the config of sphinx to change the default meaning of single backticks. Thus both single and double backticks are equivalent and occurrences of double backticks are from the older code

Reply:
Thanks @glemaitre  for the explanation :nerd_face: . I will try to follow it from now on.